hooks/pre/30-upgrade.rb

```diff
@@ -65,8 +65,8 @@ def fail_and_exit(message)
 
   if Kafo::Helpers.module_enabled?(@kafo, 'katello') || @kafo.param('capsule', 'pulp').value
     upgrade_step :start_mongo
-    upgrade_step :start_httpd
     upgrade_step :migrate_pulp
+    upgrade_step :start_httpd
```

Comment:
Do we restart other pulp services? Looking at the Pulp upgrade instructions we should be restarting workers/resource_manager/celerybeat after migrating as well.